release.md

```diff
@@ -0,0 +1,167 @@
+# Ignition Release Process
+
+This documentation covers the releasing process done by the members of the
+Ignition/Gazebo team. It offers a general overview and documentation of the
+procedures and tools used and provide an step by step guide to do released.
+
+#### Table of Contents
+
+[General Overview](#general-overview)
```

Comment:
I think these links won't work, see how the `contributing` page is broken:

https://ignitionrobotics.org/docs/all/contributing

Reply:
I see. I think I assumed that they will work the same way it happens in github but the href seems wrong. Should I remove them or we want another implementation?

Reply:
I don't know if we have any working examples :/

Reply:
removed 2fccb23